inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -130,11 +132,61 @@ void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
     }
 }
 
+NodeVector get_ordered_ops(const ngraph::Function& f) {
+    NodeVector op_order;
+    std::stack<std::shared_ptr<Node>> stack;
```

Comment:
It's not recommended to use type names as variable names. If for any reason somebody adds `using namespace std` to this file, this will cause compilation errors.

Reply:
Applied in [this](https://github.com/openvinotoolkit/openvino/pull/3156/commits/1b80d2856153c7ccad62f8cd9146934a34529b23) commit.